models/Non-linear Cart-Pole/Cart-Pole.jl

```diff
@@ -1,3 +1,33 @@
+# # Non-linear Cart-Pole
+#
+# The model was a version of the cart-pole introduced by Barto, Sutton, and Anderson in [xxx].
+
+# ## Model
+#
+# The dynamics of the cart-pole system are described as follows:
+#
+# ```math
+# \begin{aligned}
+# \ddot{x} &= \dfrac{u + mlω^2*sin(θ)}{mt} - \dfrac{ml(g sin(θ)- cos(θ))
+#             (\dfrac{u + mlω^2*sin(θ)}{mt})}{l(\dfrac{4}{3} - \dfrac{m cos(θ)^2}{mt})}
+#             \dfrac{cos(θ)}{mt}    \\
+# \ddot{θ} &= \dfrac{g sin(θ)- cos(θ)(\dfrac{u + mlω^2*sin(θ)}{mt})}{l(\dfrac{4}{3}
+#             - \dfrac{m cos(θ)^2}{mt})} \dfrac{cos(θ)}{mt}
+# \end{aligned}
+# ```
+# where ``u ∈ {−10,10}`` is the input force, which either pushes the cart left
+# or right, ``g = 9.8`` is gravity, ``m = 0.1`` is the pole’s mass, ``l = 0.5``
+# is half the pole’s length , ``mt = 1.1`` is the total mass, ``x`` is the
+# position of the cart, θ is the angle of the pendulum with respect to the
+# positive y-axis, ``v = ̇x`` is the linear velocity of the cart, and
+# ``ω = ̇θ`` is the angular velocity of the pendulum. The controller has four
+# inputs ``(x, ̇x, θ, ̇θ)``, four layers with ``[24,48,12,2]`` neurons
```

Comment:
the dot doesn't look in th correct position.. i would suggest to use ``\dot{x}`` etc